napari_animation/_qt/animation_widget.py

```diff
@@ -40,37 +40,42 @@ def __init__(self, viewer: Viewer, parent=None):
         self.viewer = viewer
         self.animation = Animation(self.viewer)
 
-        # Initialise UI
-        self._init_ui()
+        # Initialise User Interface
+        self.keyframesListControlWidget = KeyFrameListControlWidget(
+            animation=self.animation, parent=self
+        )
+        self.keyframesListWidget = KeyFramesListWidget(
+            self.animation.key_frames, parent=self
+        )
+        self.frameWidget = FrameWidget(parent=self)
+        self.saveButton = QPushButton("Save Animation", parent=self)
+        self.animationsliderWidget = AnimationSliderWidget(
+            self.animation, orientation=Qt.Horizontal, parent=self
+        )
+
+        # Create layout
+        self.setLayout(QVBoxLayout())
+        self.layout().addWidget(self.keyframesListControlWidget)
+        self.layout().addWidget(self.keyframesListWidget)
+        self.layout().addWidget(self.frameWidget)
+        self.layout().addWidget(self.saveButton)
+        self.layout().addWidget(self.animationsliderWidget)
```

Comment:
glad you agree.  I'd say the main exception to this would be if you find that you need to _re_-initialize things in the lifetime of the widget.  Then, it makes sense to pull that out into a method that can be called.

Reply:
Yeah, agreed i like it too, thanks